gson/src/main/java/com/google/gson/TypeAdapterFactory.java

```diff
@@ -22,7 +22,8 @@
  * Creates type adapters for set of related types. Type adapter factories are
  * most useful when several types share similar structure in their JSON form.
  *
- * <h2>Example: Converting enums to lowercase</h2>
+ * <h2>Examples</h2>
```

Comment:
Have added a new parent heading here to group the examples a bit, I hope that is fine.